.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = { "extends": "standard" };
```

Comment:
Just curious... is this for IDEs/VScode?

Reply:
I added it so VScode would highlight / fix lint errors. I thought the purpose of an .eslintrc file is just setting custom linting rules, but I couldn't get VScode to catch errors without it. I can pop it out if you don't want to add it at this point.

Reply:
Got it, no that's fine. Thanks.